PowerEditor/installer/nativeLang/portuguese.xml

```diff
@@ -630,14 +630,16 @@ Nota de tradução:
 					<Item id="2219" name="Palavras-chave predefinidas"/>
 					<Item id="2221" name="Palavras-chave do utilizador"/>
 					<Item id="2225" name="Linguagem:"/>
-					<Item id="2226" name="Ativar cor do primeiro plano global"/>
-					<Item id="2227" name="Ativar cor do fundo global"/>
-					<Item id="2228" name="Ativar tipo de letra global"/>
-					<Item id="2229" name="Ativar tamanho do tipo de letra global"/>
-					<Item id="2230" name="Ativar estilo do tipo de letra global"/>
-					<Item id="2231" name="Ativar estilo do tipo de letra Itálico global"/>
-					<Item id="2232" name="Ativar estilo do tipo de letra sublinhada global"/>
-					<Item id="2234" name="Ir para as definições"/>
+					<Item id="2226" name="Forçar cor de primeiro plano para todos os estilos"/>
+					<Item id="2227" name="Forçar cor de plano de fundo para todos os estilos"/>
+					<Item id="2228" name="Forçar tipo de letra para todos os estilos"/>
+					<Item id="2229" name="Forçar tamanho de tipo de letra para todos os estilos"/>
+					<Item id="2230" name="Forçar tipo de letra em negrito para todos os estilos"/>
+					<Item id="2231" name="Forçar tipo de letra em itálico para todos os estilos"/>
+					<Item id="2232" name="Forçar tipo de letra sublinhada para todos os estilos"/>
+					<Item id="2234" name="Ir para definições"/>
+					<!-- Don't translate "&quot;Global override&quot; -->
+					<Item id="2235" name="O que é Substituição global?"/>
```

Comment:
"Global override" should not be translated.

Reply:
Done. Thank you!